core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.ts

```diff
@@ -31,7 +32,9 @@ oppia.directive('audioTranslationBar', [
         $('.oppia-translation-tab').on('dragover', function(evt) {
           evt.preventDefault();
           scope.dropAreaIsAccessible = GLOBALS.can_translate;
-          scope.userIsGuest = !GLOBALS.userIsLoggedIn;
+          UserService.getUserInfoAsync().then(function(userInfo) {
```

Comment:
Why not check this just once, at the outset, rather than on every event call? It's not going to change throughout the session.

/cc @DubeySandeep